lib/resty/htmlentities.lua

```diff
@@ -0,0 +1,64 @@
+local ffi = require('ffi')
```

Comment:
Is this file meant to be committed in, rather than coming from the submodule?

Reply:
Yep you are right. Have removed and added to .gitignore